### DISCUSSION TITLE
😒Reject on my application

### DISCUSSION THREAD

**Discussion by jyan-R** [community/community#52029] [GitHub Education]
My credentials were denied more than 10 times. I've followed the requirement and uploaded materials including my school ID with date, my transcrpit, my enrollment letter, and any other official documentation as far as i can provide.

**Comment by Ntshangase** [comment]
Hey @jyan-R 

It would be easier to help you if you provide us the rejection reason.

**Reply by jyan-R** [reply]
I've followed the suggestion given by the rejection reason but it still not works. (i've completed my profile and tried again, and after trying i changed it back for privacy) The reason is as follow:

---
Unfortunately, we weren't able to approve your educational discount request for [@jyan-R](https://realstudent.plz).

**What happened?**
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

You are unlikely to be verified until you have completed your [GitHub user profile](https://helpmegetin.plz) with your full name as it appears in your academic affiliation document plus a short bio.

Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

**Reply by Ntshangase** [reply]
- Make sure that the Student ID Card you send is dated to show that you are registered for this year.
- The name on your GitHub profile is the same as the one on the submitted documents.

**Comment by samus-aran** [comment]
Hi @jyan-R 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance

**Reply by jyan-R** [reply]
😢Thank you for your help. I submitted a live photo with legible text but was still denied with the following reason:

<img width="503" alt="image" src="https://user-images.githubusercontent.com/87905051/230599475-971c7ca4-80ee-43c0-b55e-25cd2a9e93b5.png">

**Reply by samus-aran** [reply]
Hi @jyan-R 

Thank you for providing a live photo 👍 Here are some other tips regarding to translation.

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- **We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.**

Let us know how you get on 👍

**Reply by elilau755** [reply]
I have encountered the same problem, whether it is the transcript (including name, student ID, Github username, current year and time, school name and grades), our education online verification report in China (also including name, enrollment date, expected graduation date, and school attended), or the student card, student ID, all of the above materials I have provided are in English. In addition, my Github profile has long been filled out according to the requirements, but no matter how many times I take photos and upload them on my phone, I always receive the same response.

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 13, 2023 for [@elilau755](https://github.com/elilau755).

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.


Once you have resolved the issues with your sumbission, [visit us again and reapply](https://education.github.com/).

Have questions? Join the [GitHub Education Community](https://github.com/orgs/community/discussions/categories/github-education) for answers.

💖 The GitHub Education Team


@samus-aran @Ntshangase @pnpn111

**Reply by CMakey** [reply]
same question, i saw one solution that suggest getting the transcript with official stamp.  but my school didn't prove this for ### student, maybe u can try it.

**Reply by pnpn111** [reply]
@elilau755 
> Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

![](https://pic4.zhimg.com/80/v2-4b74dd6c3de068777cec37ee81a21e3f_720w.webp)
(Photo from Internet)

Do you Take a picture of an uploaded file every time by using the "take a picture" on your GitHub page on your phone?

> You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Please take a closer look at [my article](https://www.luogu.com.cn/blog/pnpn111/github-student-packet) on this issue. As far as I can see, the personal data you provided is insufficient and more information is needed, such as:

* You have not uploaded your profile picture, so it is recommended to directly use your student ID profile picture (take a photo of it and upload it to GitHub).

* The location information provided should be accurate to the city and should be consistent with the location of your school.

* Change your name to the English translation of your real name on your student ID card.

* Fill in more information, including the English name of your school (which should be the same as the English name you entered when submitting the application) and the year of enrolment.

* Provide a link to the school's official website.

* **Please note that after updating the information, please be sure to log out and log in again before applying.**

See my article for other details. If you still can't solve the problem, I don't know what to do. You may need to make a separate post asking for forum management help.

**Reply by CMakey** [reply]
@pnpn111 hello, when i get into GitHub ,the net is really bad, so i use vpn to accelerate it, but it stroke in uploading after taking picture ,what should i do, would be more than appreciate if u can give me some suggest.

**Reply by pnpn111** [reply]
@jisidehuli 
I'm sorry that I can't provide you with other solutions, because GitHub's automatic review system will detect whether your visiting IP is in the same city as the school you fill in. Please try not to use proxy to visit (the longest page loading time when I applied was up to one minute, but I still didn't use proxy). Maybe you can try another network (as long as it's in the same city), but if that doesn't work, you might want to ask the forum management for help.

**Reply by CMakey** [reply]
@pnpn111  ok, thank u! it cost me more than almost 5min to uploading.😵‍💫. by the way, thanks for u blog, it helps me in preparing proofs.

**Reply by pnpn111** [reply]
@jisidehuli 
You're welcome! In fact, I've been stuck with the GitHub student package registration problem for over a week, and it took me a lot of information to solve it, so I think it's necessary to write a blog about some of the details and conclusions that are difficult to get easily. I'm glad the blog can help you!

**Reply by CMakey** [reply]
@jyan-R ,i think u can use methods at @pnpn111 's blog , i think update u profile and use the bilingual proofs maybe works. i even download the english version from chai.com and use camera to capture pics, but it doesn't  work.  
there is my solution while stroking in upload pic: i use vpn just in uploading, i think it doesn't matter, because the pics couldn't send to amazon's.

**Comment by pnpn111** [comment]
Hey!
I have also been troubled by this problem for a long time, and finally solved it in the forum management prompts and my personal analysis and summary. The following article is my summary of the solution to this problem, and I hope it will be helpful to you.
[Github 学生包注册注意](https://www.luogu.com.cn/blog/pnpn111/github-student-packet)
